crates/forge/tests/e2e/plugin_diagnostics.rs

```diff
@@ -0,0 +1,439 @@
+use crate::e2e::common::runner::setup_package_at_path;
+use camino::Utf8PathBuf;
+use indoc::{formatdoc, indoc};
+use scarb_api::ScarbCommand;
+use shared::test_utils::output_assert::{assert_stdout_contains, case_assert_stdout_contains};
+use snapbox::cmd::Command as SnapboxCommand;
+use std::fs;
+
+#[test]
+#[allow(clippy::too_many_lines)]
+fn syntax() {
+    let temp = setup_package_at_path(Utf8PathBuf::from("diagnostics/syntax"));
+    let output = SnapboxCommand::from_std(
+        ScarbCommand::new()
+            .current_dir(&*temp)
```

Comment:
```suggestion
            .current_dir(temp.path())
```
Also in other places